components/eamxx/mkdocs.yml

```diff
@@ -4,10 +4,9 @@ nav:
   - 'Home': 'index.md'
   - 'User Guide':
     - 'Overview': 'user/index.md'
-    - 'Installation': 'common/installation.md'
-    - 'Model output': 'user/model_output.md'
+    - 'EAMxx case basics': 'user/eamxx_cases.md'
     - 'Model input': 'user/model_input.md'
-    - 'Runtime parameters': 'common/eamxx_params.md'
```

Comment:
We already link this page from the "Model input" page.

Reply:
Yeah, I actually hate the fact that we have a "common" documentation subdirectory at all. We should only document any particular thing in one place and refer to that page via link if needed elsewhere.